plex_trakt_sync/media.py

```diff
@@ -13,13 +14,17 @@ class Media:
     Class containing Plex and Trakt media items (Movie, Episode)
     """
 
-    def __init__(self, plex, trakt, plex_api: PlexApi = None, trakt_api: TraktApi = None):
+    def __init__(self, plex: PlexLibraryItem, trakt, plex_api: PlexApi = None, trakt_api: TraktApi = None):
         self.plex_api = plex_api
         self.trakt_api = trakt_api
         self.plex = plex
         self.trakt = trakt
         self.show = None
 
+    @nocache
```

Comment:
remove @nocache here, it should be on actual method making the call, i.e fetch_item